FantasyUniverse/Views/App views/UserInfoView.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  UserInfoView.swift
+//  FantasyUniverse
+//
+//  Created by TarasPeregrinus on 31.07.2022.
+//
+
+import SwiftUI
+
+struct UserInfoView: View {
+    
+    var user: User
+    @Binding var userImage: Image?
+    @State private var showingAvatar = false
+    let logoutAction: () -> Void
+    let editProfileAction: () -> Void
+    let imageTappedAction: () -> Void
```

Comment:
Looks complex enough for a view model if you ask me 🤔

Reply:
Hmm... I will probably add a viewModel, but later, as there is be more logic planned.